x-pack/plugins/apm/ftr_e2e/cypress/e2e/settings/agent_configurations.cy.ts

```diff
@@ -123,7 +123,7 @@ describe('Agent configuration', () => {
       .click({ force: true })
       .type('All');
 
-    cy.get('mark').contains('All').click();
+    cy.get('mark').contains('All').click({ force: true });
```

Comment:
Is this intentional?

Reply:
Yes, i found this test failing sometimes. With this we remove the flakiness